frontend/src/metabase/components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/display-name */
```

Comment:
Nit: I think we can avoid the eslint-disable by using using the following pattern (taken from [`DateInput.stories.tsx`](https://github.com/metabase/metabase/blob/d99d06a237baf499aa92bb4cd4a721a6825a26bc/frontend/src/metabase/core/components/DateInput/DateInput.stories.tsx#L11)):
```
const Template: ComponentStory<typeof DateInput> = args => {
  const [value, setValue] = useState<Moment>();
  return <DateInput {...args} value={value} onChange={setValue} />;
};

export const Default = Template.bind({});
```